src/components/QuestionPage.js

```diff
@@ -58,43 +66,62 @@ class QuestionPage extends Component {
       if (this.state.answer == this.state.correctAnswer) {
         return {
           backgroundColor: '#58E778',
-          height: 32,
           width: 294,
           justifyContent: 'center',
           alignItems: 'center',
-          marginBottom: 21,
+          marginBottom: 10,
           flexDirection: 'row',
         };
       } else {
         return {
-          backgroundColor: '#F16B8B',
-          height: 32,
+          backgroundColor: '#ec4646',
           width: 294,
           justifyContent: 'center',
           alignItems: 'center',
-          marginBottom: 21,
+          marginBottom: 10,
           flexDirection: 'row',
         };
       }
     } else {
-      return {
-        backgroundColor: '#6BB1F1',
-        height: 32,
-        width: 294,
-        justifyContent: 'center',
-        alignItems: 'center',
-        marginBottom: 21,
-        flexDirection: 'row',
-      };
+      if (
+        item == this.state.disableAnswer1 ||
+        item == this.state.disableAnswer2
```

Comment:
[this.state.disableAnswer1,this.state.disableAnswer2].includes(item)

Reply:
Style ları böyle if conditionlarına koymak doğru bir pratik değil gibi

Reply:
Burada sadece değişen backgroud color bunu StyleSheet.create ile yapmak lazım. 

style={[styles.answer, {backgroundColor:koşul?"#fff":"red"}]} 
Gibi

Reply:
> Burada sadece değişen backgroud color bunu StyleSheet.create ile yapmak lazım.
> 
> style={[styles.answer, {backgroundColor:koşul?"#fff":"red"}]}
> Gibi

koşul yapısı birazcık karmaşık olduğu için yine if else ile başka bir fonksiyon içinde returnlemek zorunda kaldım. Ama diğer style ları tek hale getirip sadece backgroundColor'u if-else'e bağladım. `style={[
                this.buttonJoker(),
                {
                  height: 30,
                  width: 100,
                  justifyContent: 'center',
                  alignItems: 'center',
                  borderRadius: 5,
                },
              ]}>`